content/themes/casper/default.hbs

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html>
 <head>
+    <script> window.loadmillUserId = "21" </script>
```

Comment:
Did you split the user id and registration script between the header and body on purpose? 
Intercom for example recommends to add them both before the closing body tag - [link](https://www.intercom.com/help/install-intercom-on-your-product-or-site/quick-install/install-intercom-on-your-web-app-for-logged-in-users)

Reply:
Not sure it matters. Will move it.